### PR TITLE
Upload flow - fixing created time change

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33480

## Description
When a pack has only single section in changelog - the initial one, and it's updated without bumping a version (e.g. description change), the creation time will change to the updated time.

### CODE REVIEW

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -1575,22 +1575,43 @@ def _get_changelog(self, index_folder_path):
...
+        Returns:
+            dict: Get the metadata from downloaded index
+        """
+        metadata_index_path = os.path.join(index_folder_path, self._pack_name, Pack.METADATA)
+        metadata = {}
+        if os.path.exists(metadata_index_path):
+            with open(metadata_index_path, "r") as metadata:
+                metadata = json.load(metadata)
+        return metadata
+
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:1591]: let's construct the path and use `load_json` instead

**Review by guyfreund — CHANGES_REQUESTED** [review]

**Review by guyfreund — APPROVED** [review]
Looking good

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -1561,36 +1561,21 @@ def format_metadata(self, user_metadata, pack_content_items, integration_images,
...
         """ Gets the pack created date.
         Args:
             index_folder_path (str): downloaded index folder directory path.
         Returns:
             datetime: Pack created date.
         """
-        earliest_changelog_released_date = datetime.utcnow().strftime(Metadata.DATE_FORMAT)
-        changelog = self._get_changelog(index_folder_path)
+        created_time = datetime.utcnow().strftime(Metadata.DATE_FORMAT)
+        metadata = load_json(os.path.join(index_folder_path, self._pack_name, Pack.METADATA))
```
Comment (yuvalbenshalom) [inline Tests/Marketplace/marketplace_services.py:1572]: If the file dose not exist, I think this can create exception. no?

Comment (ohaim1008) [reply Tests/Marketplace/marketplace_services.py:1572]: we check it in `load_json`

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**